cloudant-client/src/main/java/com/cloudant/client/api/Database.java

```diff
@@ -944,6 +944,33 @@ public boolean contains(String id) {
     }
 
     /**
+     * Reads an attachment from the database.
+     *
+     * The stream must be closed after usage, otherwise http connection leaks will occur.
+     *
+     * @param docId the document id
+     * @param attachmentId the attachment name
+     * @return the attachment in the form of an {@code InputStream}.
+     */
+    public InputStream getAttachment(String docId, String attachmentId) {
```

Comment:
Depending on the decision for the underlying lightcouch Database, this should call into `getAttachment(String, String, String)`.